app/views/layouts/base.html.erb

```diff
@@ -47,11 +47,12 @@
     <![endif]-->
     <% if User.current %>
       <% cache(TopbarSweeper.fragment_name, :skip_digest => true) do %>
-        <%= render "home/topbar" %>
-        <%= render "home/navbar" %>
+        <div id="layout"></div>
       <% end %>
       <div class="container-fluid container-pf-nav-pf-vertical nav-pf-persistent-secondary">
-        <%= yield(:content) %>
+      <%# mounting layout after the container for fixing the body width after a full page reload %>
```

Comment:
please don't add comments into erb, this causes hard to debug problems when we extend the page through deface

Reply:
why? (its the first time I hear about it) - can you elaborate please? and what would be the proper way to add code comments in this case?

Reply:
please see https://github.com/theforeman/foreman/pull/3453 and https://projects.theforeman.org/issues/14769/#note-4 for details

I don't know if there's already a fix in libxml and libxslt these days, but I think it's just safer to avoid